### DISCUSSION TITLE
Envío de certificados en lote

### DISCUSSION THREAD

**Discussion by PabloLis** [kleer-la/eventer#89] [Ideas]
Hola Juan,
pruebo comentar y solicitar algo por aquí (mi primera vez).
Desde un cliente nos pidieron que el envío de certificados se haga también al área de Capacitación de TH. Lo quisieran por temas de actualización de perfiles, trazabilidad etc. Creo que no es la primera vez que, al hacer cursos InCompany, alguien de RRHH nos pide que le enviemos copia o todos los certificados a ellos para ellos luego reenviarlo a los participantes. 
¿Te suena el pedido?
Si suena podríamos explorar posibles soluciones.
Gracias
Pablo

**Comment by jgabardini** [comment]
Gracias por iniciar la conversación x acá 😄 

Los workaround actuales son:
- emitir todos los certificados para los asistentes (les llega a sus mails) y luego cambiar los mails para enviarlos a RRHH
- emitirlos uno por uno guardarlos y luego enviarlos a RRHH
- cargar nuevamente los asistentes, con mail de RRHH (quedarían 2 veces cada asistentes, una vez con mail propio otra con mail de RRHH)

Habiendo formas de realizarlo, no estaría entre los próximos cambios. Veamos si es algo frecuente de acá en adelante, en cuyo caso reevaluamos.
